### PR TITLE
Update Enos version to 0.0.14

### PR DESCRIPTION
**Pull request by manthonygfp**
### How to read this pull request
We have cut a new release of Enos, and this PR updates the default version of Enos that is set up by the action.

I also added a .gitignore file so that node_modules provisioned locally would not be accidentally committed.

### Checklist
- [x] The commit message includes an explanation of the changes
- [x] Manual validation of the changes have been performed (if possible)
- [x] New or modified code has requisite test coverage (if possible)
- [x] I have performed a self-review of the changes
- [x] I have made necessary changes and/or pull requests for documentation
- [x] I have written useful comments in the code

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,5127 +1,8 @@
 {
   "name": "action-setup-enos",
-  "version": "0.0.12",
-  "lockfileVersion": 2,
+  "version": "0.0.14",
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```
Comment (ryancragun) [inline package-lock.json:6]: Weird, this seems like a yuuuuge diff

**Review by ryancragun — APPROVED** [review]
I'm a little spooked about the massive change to the package lock but I suspect it's just a different version of npm doing things.

**Review by mikebaum — APPROVED** [review]
looks good to me

File: package-lock.json (hunk 1 above)
Comment (manthonygfp) [reply package-lock.json:6]: yeah...

Comment (manthonygfp) [reply package-lock.json:6]: I did run a second npm install after seeing this comment and ended up with the same results. For posterity, this was done with npm version `6.14.17`.

**Pull request merged**